app/logging.py

```diff
@@ -53,19 +54,24 @@ def _configure_watchtower_logging_handler():
             region_name=aws_region_name,
         )
 
-        root = logging.getLogger()
-        handler = watchtower.CloudWatchLogHandler(
-            boto3_session=boto3_session,
-            log_group=log_group,
-            stream_name=stream_name,
-            create_log_group=create_log_group,
-        )
-        handler.setFormatter(logstash_formatter.LogstashFormatterV1())
-        root.addHandler(handler)
+        global session_boto3
+        session_boto3 = boto3_session
     else:
         print("Unable to configure watchtower logging.  Please verify watchtower logging configuration!")
 
 
+def _get_cloudwatch_handler():
+    global session_boto3
+    group = os.getenv("AWS_LOG_GROUP", "platform")
+    stream = os.getenv("AWS_LOG_STREAM", _get_hostname())
+    create_log_group = str(os.getenv("AWS_CREATE_LOG_GROUP")).lower() == "true"
+    handler = watchtower.CloudWatchLogHandler(
+        boto3_session=session_boto3, log_group=group, stream_name=stream, create_log_group=create_log_group
+    )
```

Comment:
Let’s keep environment variable retrieval in one place, the closest to the module entry point as possible.

```suggestion
def _get_cloudwatch_handler(
    aws_access_key_id, aws_secret_access_key, aws_region_name, aws_log_group, aws_stream_name, create_log_group
):
    print(f"Configuring watchtower logging (log_group={aws_log_group}, stream_name={aws_stream_name})")
    boto3_session = Session(
        aws_access_key_id=aws_access_key_id,
        aws_secret_access_key=aws_secret_access_key,
        region_name=aws_region_name,
    )

    handler = watchtower.CloudWatchLogHandler(
        boto3_session=boto3_session,
        log_group=aws_log_group,
        stream_name=aws_stream_name,
        create_log_group=create_log_group
    )
```